index.js

```diff
@@ -45,7 +45,8 @@ export function unherit(Super) {
   /**
    * Constructor accepting variadic arguments.
    *
-   * @returns {any}
+   * @class
+   * @this {Class}
```

Comment:
I guess `this` is *potentially* an instance here? So should it use `Class`? 🤔

Reply:
Maybe :man_shrugging: 
Updated this, give the new version a look